apps/docs/components/server/plugins/code-snippets.ts

```diff
@@ -21,16 +20,8 @@ export default defineNitroPlugin((nitro: any) => {
         const normalizedSrc = filename.replace(/@/g, '.');
         const nameExt = normalizedSrc.split('.').pop().toLowerCase();
 
-        let filePath = resolver.resolve(normalizedSrc);
-        if (import.meta.env.NODE_ENV === 'prerender') {
-          filePath = path.join(path.dirname(import.meta.url), '..', '..', '..', '..', filePath).replace('file:/', '/');
-          if (!fs.existsSync(filePath)) {
-            filePath = path
-              .join(path.dirname(import.meta.url), '..', '..', '..', '..', '..', resolver.resolve(normalizedSrc))
-              .replace('file:/', '/');
-          }
-        }
-
+        let filePath = resolve(resolve(),  normalizedSrc.replace('../../../../', '../../'))
```

Comment:
The `nuxt/kit` resolver has a different base path that was previously being hardcoded at build time. Since I had to switch dependencies the new resolver has a different root, so to keep the backwards compatibility and not change all of the markdown this is needed. The other option is to change all of the `<<< ../../....` in the `md` files.

Reply:
Ah okay, thank you for explanation :)